frontend/src/modules/member/member-export-limit.js

```diff
@@ -13,6 +14,9 @@ const exportMax = {
  * @returns maximum number of exports
  */
 export const getExportMax = (plan) => {
+  if (plan === Plans.values.scale) {
+    return exportMax.scale;
```

Comment:
Just thinking that we probably don't need to have an if for each plan. We could just have
`return exportMax[plan]`

Reply:
its something to do in near future to not change on so many places and have everything under one configuration